aspnetcore/security/cors.md

```diff
@@ -37,27 +37,29 @@ These URLs have different origins than the previous two:
 > [!NOTE]
 > Internet Explorer doesn't consider the port when comparing origins.
 
-## Setting up CORS
+## Enable CORS
 
+::: moniker range="<= aspnetcore-1.1"
 To set up CORS for your application add the `Microsoft.AspNetCore.Cors` package to your project.
+::: moniker-end
 
-Add the CORS services in Startup.cs:
+Call [AddCors](/dotnet/api/microsoft.extensions.dependencyinjection.corsservicecollectionextensions.addcors) in `Startup.ConfigureServices`:
 
 [!code-csharp[](cors/sample/CorsExample1/Startup.cs?name=snippet_addcors)]
 
 ## Enabling CORS with middleware
 
-To enable CORS for your entire application add the CORS middleware to your request pipeline using the `UseCors` extension method. Note that the CORS middleware must precede any defined endpoints in your app that you want to support cross-origin requests (ex. before any call to `UseMvc`).
+To enable CORS for your entire application add the CORS middleware to your request pipeline using the `UseCors` extension method. The CORS middleware must precede any defined endpoints in your app that you want to support cross-origin requests (ex. before any call to `UseMvc`).
```

Comment:
Comma after "application," and I've been using "app" (almost) everywhere. "in your app" is probably unnecessary. Possibly "where" over "that." "For example" for "ex."

> To enable CORS for your entire app, add the CORS middleware to your request pipeline using the \`UseCors\` extension method. The CORS middleware must precede any defined endpoints where you want to support cross-origin requests (for example, before any call to \`UseMvc\`).